changelog.d/20230301_153001_nedbat_edx_cla.rst

```diff
@@ -0,0 +1,6 @@
+.. A new scriv changelog fragment.
+
+- The "internal" setting is being replaced by an "internal-ghorgs" list on an
+  institution.  A pull request is now internal if the author is internal to the
+  GitHub organization.  The old "internal" setting is still used, but we'll be
+  deleting it once the new code is in place.
```

Comment:
```suggestion
- The "internal" setting is being replaced by an "internal-ghorgs" list on an
  institution.  A pull request is now internal if the author's associated org has
  the org the PR is being made to as an internal org.  The old "internal" setting 
  is still used, but we'll be deleting it once the new code is in place.
```

Reply:
I'll take this with another tweak since we use "org" in too many ways.